stdlib/gettext.pyi

```diff
@@ -113,33 +114,91 @@ def find(domain: str, localedir: StrPath | None = ..., languages: Iterable[str]
 _T = TypeVar("_T")
```

Comment:
The docs for `gettext.translations` say:

> Return a *Translations instance based on the domain, localedir, and languages, which are first passed to [find()](https://docs.python.org/3/library/gettext.html#gettext.find) to get a list of the associated .mo file paths. Instances with identical .mo file names are cached. The actual class instantiated is class_ if provided, otherwise [GNUTranslations](https://docs.python.org/3/library/gettext.html#gettext.GNUTranslations). The class’s constructor must take a single [file object](https://docs.python.org/3/glossary.html#term-file-object) argument. If provided, codeset will change the charset used to encode translated strings in the [lgettext()](https://docs.python.org/3/library/gettext.html#gettext.NullTranslations.lgettext) and [lngettext()](https://docs.python.org/3/library/gettext.html#gettext.NullTranslations.lngettext) methods.

For me, that implies that the expectation is that the value passed to the `class_` parameter should be a subclass of `NullTranslations` (`gettext.translations` won't return a `*Translations` instance in all cases otherwise).

```suggestion
_T = TypeVar("_T", bound=NullTranslations)
```

Reply:
Thanks, done. The implementation also calls the `add_fallback` method on the returned object.